### PR TITLE
support multi video stream info and add camera cap flags

### PR DESCRIPTION
**Pull request by tbago**
This PR contain the following features:

- The camera proto video stream info contain more detail about the stream info, so the camera server proto need use the same video stream info define;
- the camera can have multi lens (e.g. one Visible light one Infrared). So need support multi video stream info;
- the camera cap flags is important for clint to check the camera capability;
- unify the camera information int camera and camera server;

### CODE REVIEW

**Comment by tbago** [comment]
The pull request to MAVSDK will come soon

File: protos/camera/camera.proto [hunk 1]
```diff
@@ -421,11 +422,50 @@ message SettingOptions {
...
+                // video streaming info)
+        CAMERA_CAP_FLAGS_HAS_TRACKING_POINT =
+            9;  // Camera supports tracking of a point on the camera view.
+        CAMERA_CAP_FLAGS_HAS_TRACKING_RECTANGLE =
+            10;  // Camera supports tracking of a selection rectangle on the
+                 // camera view.
+        CAMERA_CAP_FLAGS_HAS_TRACKING_GEO_STATUS =
+            11;  // Camera supports tracking geo status
+                 // (CAMERA_TRACKING_GEO_STATUS).
+    };
```
Comment (julianoes) [inline protos/camera/camera.proto:453]: The capture flags should only be internally used. They should not be visible to the user. And if we need to expose specific ones, then they have to just be boolean flags.

File: protos/camera/camera.proto [hunk 2]
```diff
@@ -421,11 +422,50 @@ message SettingOptions {
...
+    float horizontal_sensor_size_mm = 5;  // Horizontal sensor size
+    float vertical_sensor_size_mm = 6;    // Vertical sensor size
+    uint32 horizontal_resolution_px =
+        7;                              // Horizontal image resolution in pixels
+    uint32 vertical_resolution_px = 8;  // Vertical image resolution in pixels
+    uint32 lens_id = 9;                 // Lens ID
+    uint32 definition_file_version =
+        10;  // Camera definition file version (iteration)
+    string definition_file_uri =
+        11;  // Camera definition URI (http or mavlink ftp)
```
Comment (julianoes) [inline protos/camera/camera.proto:468]: This is also only to be used internally.

File: protos/camera/camera.proto [hunk 3]
```diff
@@ -190,7 +190,7 @@ message ModeResponse {
 
 message SubscribeVideoStreamInfoRequest {}
 message VideoStreamInfoResponse {
-    VideoStreamInfo video_stream_info = 1; // Video stream info
+    repeated VideoStreamInfo video_stream_infos = 1; // Video stream infos
```
Comment (julianoes) [inline protos/camera/camera.proto:193]: How does this work when it's repeated? Do you get multiple video stream infos, one per video stream?

**Review by julianoes — CHANGES_REQUESTED** [review]

**Pull request closed without merging**